### ISSUE TITLE
 No Google Analytics: Library Incompatible. Caused when using GoogleAdMobAdsSdk

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#279]
```
---------------------------------------------------------------------------
NOTE: This issue tracking system is for Google Analytics developer products
only.
If you are not a developer/programmer visit:
http://www.google.com/analytics/support.html
---------------------------------------------------------------------------
Name of affected component: Mobile Tracking

Name of related library and version, if applicable (e.g. Android, iOS,
Snippets, etc.):
Android - libGoogleAnalyticsV2.jar

Issue summary:
when I also use GoogleAdMobAdsSdk-6.3.1.jar, the following error info printed:
03-24 15:26:21.777: W/Ads(28309): No Google Analytics: Library Incompatible.

After I removed GoogleAdMobAdsSdk-6.3.1.jar from libs, this warning gone. seems 
ok.

Steps to reproduce issue:
1. Use the sample project to reproduce this issue.
2.
3.

Expected output:
No such warnng.

Actual results:
What do you actually see after performing the above steps?

Notes:
My test android device is 4.0.4 version.
```

Original issue reported on code.google.com by `kouxing2...@gmail.com` on 24 Mar 2013 at 7:33

Attachments:
- [AndroidTest.zip](https://storage.googleapis.com/google-code-attachments/analytics-issues/issue-279/comment-0/AndroidTest.zip)

**Comment by GoogleCodeExporter** [comment]
```
Same problem here with latest analytics 2.0-beta5 and Google Admob SDK 6.4.1. 
But not just a warning occured, complete analytics for Android app is broken. 
After removing the admob sdk the warning is gone and mobile analytics works 
again. 
```

Original comment by `christl....@gmail.com` on 1 May 2013 at 1:19

**Comment by GoogleCodeExporter** [comment]
```
Same "No Google Analytics: Library Incompatible" problem here with 
libGoogleAnalyticsV2.jar of Analytics 2.0-beta5 and 
GoogleAdMobAdsSdk-6.4.1.jar, on HTC One with Android 4.1.1. Any decent 
workaround?
```

Original comment by `seeingwi...@gmail.com` on 6 Jun 2013 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
Hmmm, so actually in my case there was no problem. It was delay caused by 
Analytics. I saw metric value being sent only once in Android's log (in url) 
but this is because this is per-use metric. It showed up for all events once 
they were processed.
```

Original comment by `michal.d...@gmail.com` on 11 Jun 2013 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Well, in the last few days I released my app despite this error message and I 
still see analytics results for the new version, so I'm not sure what to make 
of it. It is not or not quite that analytics does not work, despite the 
worrying "No Google Analytics" message.
```

Original comment by `seeingwi...@gmail.com` on 11 Jun 2013 at 9:57

**Comment by GoogleCodeExporter** [comment]
```
This message from the Mobile Ads SDK shouldn't impact data collection via the 
GA SDK. Closing this issue for now.
```

Original comment by `awa...@google.com` on 8 Nov 2013 at 8:12
- Changed state: **WontFix**
